### DISCUSSION TITLE
RE4 Mod....  PLEASE tell me it's coming praydog!

### DISCUSSION THREAD

**Discussion by davidflana** [praydog/REFramework#957] [General Questions]
I am not sure where I read it, but I thought I saw somewhere that Praydog was waiting for the powers that be to release the VR version of RE4 for PSVR2 and then they would release their mod a little while later.  Is this true??  PLEASE tell us you are going to release it!  We love your mods so much, we NEED to finish the saga with RE4!!!   How about for the HOLIDAYS????  a wonderful Holiday gift for your adoring fans????

**Comment by Satan-friend** [comment]
What you have written is true. Flat2VR wrote about it on Twitter. Unfortunately Praydog himself did not confirm it :/ he only uploaded the tests 9 months ago on YouTube and since then there is silence from his side.

Also looking forward to this mod. I am holding off buying the game especially for it. After the great RE 2 and 3 (I bought them for Praydog's VR mods) I can't imagine playing Resident Evil on a flat screen.

**Reply by davidflana** [reply]
I did the same!  Also bought RE7 and RE8 and played with the Praydog VR mods and they were fantastic!  Fingers crossed!

**Reply by Satan-friend** [reply]
Oh yes, I'm currently playing RE 7 in VR with Praydog's mod :D as soon as I finish I'll buy RE 8 as well. Praydog has made this series the best (just like Half-Life and Serious Sam) series of VR games. I hope we will see support for RE 4.

**Reply by davidflana** [reply]
RE 7 in VR is some scary sh!t !  so much more spooky than the others!
